code/game/gamemodes/dynamic/dynamic_rulesets_midround.dm

```diff
@@ -82,6 +82,8 @@
 		var/threat = round(mode.threat_level/10)
 		if (job_check < required_enemies[threat])
 			return FALSE
+		if(EMERGENCY_PAST_POINT_OF_NO_RETURN)
```

Comment:
It shouldn't get to this point. This should be placed in the code that actually handles the midround rolls.

Reply:
You're right, sorry.